samples/Build2019Demo.Unity/Assets/Demo/Scripts/Demo.asmdef

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Demo",
+    "references": [],
+    "optionalUnityReferences": [],
+    "includePlatforms": [
+        "Editor"
+    ],
```

Comment:
Does this indicate that the entire Scripts directory is editor-only? Maybe the directory/assembly should be named "EditorScripts"/"DemoEditor" to make this clear

Reply:
Yes, but now it's removed :)